chromatic/tests/test_visualizations.py

```diff
@@ -245,3 +245,17 @@ def test_pcolormesh():
     b.imshow(ax=ax[1, 0])
     b.pcolormesh(ax=ax[1, 1])
     plt.savefig(os.path.join(test_directory, "test-pcolormesh-vs-imshow.png"))
+
+
+def test_plot_noise_comparison():
```

Comment:
I added a test here for a simple with/without systematics dataset. This is going to be a super useful function!